tests/e2e/portal/VTK_file.js

```diff
@@ -0,0 +1,80 @@
+// node VTK_file.js [url] [file_type] [viewer_key] [viewer_vs] [download_link] [file_name] [file_size] [--demo]
+
+const tutorialBase = require('../tutorials/tutorialBase');
+const utils = require('../utils/utils');
+
+const args = process.argv.slice(2);
+const {
+  urlPrefix,
+  params,
+  enableDemoMode
+} = utils.parseCommandLineArgumentsStudyDispatcherParams(args);
+
+let anonURL = urlPrefix + "/view";
+for (let i=0; i<Object.keys(params).length; i++) {
```

Comment:
In the real scenario, the ``anonURL`` head provided by the backend upon request. 

It would be a very nice extension to this test instead of having the url hard-coded. 

@ignapas that might even be part of the code you had to put in the portal? Would you be willing to extend this e2e?

Reply:
I guess that this is where the magic happens:
https://github.com/nih-sparc/sparc-app/blob/f70241f5d16ebac346a1bd75a321b0e75b8d50fe/components/FilesTable/OsparcFileViewersDialog.vue#L92

Reply:
Will do 👍